static/less/form.less

```diff
@@ -70,6 +70,10 @@ select.form-control:not([multiple]) {
 
 label {
     font-weight: normal;
+}
+
+//.select-on-check-all:first-child
```

Comment:
@MasterHans Why is this commented out? Shouldn't that be restricting the change?

Reply:
This is debugging comment. Unfortunately its could not work on here